File: python/openai_sdk.py
import os

from openai import OpenAI


client = OpenAI(
    api_key=os.environ["INFERFLOW_API_KEY"],
    base_url=os.environ.get("INFERFLOW_BASE_URL", "https://api.inferflow.dev/v1"),
)

model = os.environ.get("INFERFLOW_MODEL", "deepseek-v4-flash")

response = client.chat.completions.create(
    model=model,
    messages=[
        {"role": "user", "content": "Reply in one sentence: Inferflow setup works."}
    ],
)

print(response.choices[0].message.content)

File: javascript/openai_sdk.mjs
import OpenAI from "openai";

const client = new OpenAI({
  apiKey: process.env.INFERFLOW_API_KEY,
  baseURL: process.env.INFERFLOW_BASE_URL ?? "https://api.inferflow.dev/v1",
});

const model = process.env.INFERFLOW_MODEL ?? "deepseek-v4-flash";

const response = await client.chat.completions.create({
  model,
  messages: [
    { role: "user", content: "Reply in one sentence: Inferflow setup works." },
  ],
});

console.log(response.choices[0].message.content);
